src/parser/priest/discipline/modules/features/Checklist/Component.tsx

```diff
@@ -14,8 +14,8 @@ import Requirement from 'parser/shared/modules/features/Checklist/Requirement';
 import PreparationRule from 'parser/shared/modules/features/Checklist/PreparationRule';
 import GenericCastEfficiencyRequirement from 'parser/shared/modules/features/Checklist/GenericCastEfficiencyRequirement';
 
-const DisciplinePriestChecklist = ({ combatant, castEfficiency, thresholds }) => {
-  const AbilityRequirement = props => (
+const DisciplinePriestChecklist = ({ combatant, castEfficiency, thresholds }: any) => {
```

Comment:
`combatant: Combatant, castEfficiency: CastEfficiency, thresholds: any` (im pretty sure I have the thresholds type somewhere added but I think it requires going through *every* one of your thresholds and making sure they are correct.. wont block this PR for that)